scripts/set-git-source.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOTDIR="${SCRIPTDIR}/.."
+[[ -n "${DEBUG:-}" ]] && set -x
+
+
+if [ -z ${GIT_ORG} ]; then echo "Please set GIT_ORG when running script, optional GIT_BASEURL and GIT_REPO to formed the git url GIT_BASEURL/GIT_ORG/*"; exit 1; fi
+
+
+GIT_HOST=${GIT_HOST:-github.com}
+GIT_BASEURL="https://${GIT_HOST}"
+GITOPS_REPO=${GITOPS_REPO:-multi-tenancy-gitops-apps}
```

Comment:
I updated the variables name at upstream bootstrap.sh to specify apps git repo `GIT_GITOPS_APPLICATIONS`  and branch `GIT_GITOPS_APPLICATIONS_BRANCH`
```
GIT_GITOPS_APPLICATIONS=${GIT_GITOPS_APPLICATIONS:-multi-tenancy-gitops-apps}
GIT_GITOPS_APPLICATIONS_BRANCH=${GIT_GITOPS_APPLICATIONS_BRANCH:-master}
```

Reply:
Is here https://github.com/cloud-native-toolkit-demos/multi-tenancy-gitops-apps/blob/master/scripts/ace-bootstrap.sh#L28
and here 
https://github.com/cloud-native-toolkit/multi-tenancy-gitops/blob/master/scripts/bootstrap.sh#L54-L61

Reply:
Thanks, updated the set-git-source.sh to use the new env var (`GIT_GITOPS_APPLICATIONS`and `GIT_GITOPS_APPLICATIONS_BRANCH`).